lib/DistributionsCore/src/DistributionsCore.jl

```diff
@@ -0,0 +1,521 @@
+module DistributionsCore
+import Base: maximum, minimum, extrema, ==
+## sample space/domain
+
+"""
+`F <: VariateForm` specifies the form or shape of the variate or a sample.
+"""
```

Comment:
What did you include in this file? Maybe keep the same file names as in Distributions to make it clearer? Maybe we want to reorganize it at some point but it might be clearer for the transition.

Reply:
This file is a copy and paste of common.jl.

Reply:
Just keep it in a separate common.jl file? I assume there might be also other things that we would like to add, and it seems cleaner to move the files than to merge them, doesn't it?

Reply:
makes sense. Will do.